packages/compass-sidebar/src/components/navigation-items.tsx

```diff
@@ -258,16 +278,21 @@ export function NavigationItems({
   );
 }
 
+const DEFAULT_SERVER_DATABASES = ['admin', 'config', 'local'];
```

Comment:
mongodb-ns package provides this metadata (and handles some other cases too that you are not covering here), would it make sense to use it instead of hardcoding this list?